Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -51,23 +53,46 @@
- BTWK_story_event.0115.desc_alliance:0 "Word has reached you that through arranging a marriage between Norwenna - the widow of the late Prince of Dumnonia, Mordred - and (SPOUSE), Uther has secured an alliance between his realm, and (NEW REALM). A number of guardians have also been named for the Edling Mordred. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. Such a web of alliances will surely fortify Dumnonia against its enemies until the young Prince reaches maturity."
- BTWK_story_event.0115.desc_arthur:0 "Word has reached you that, despite the misgivings he holds for his bastard son, a marriage has been arranged between Prince Arthur, and his late brother's widow, Norwenna. With Arthur's arrangment as the Edling Prince's foster father, his return to the shores of Britannia from the Armorican frontlines is imminent. A number of guardians have also been named for the young Mordred. Chiefly among them are Tewdric, King of Gwent, and the legendary druid Merlin. While Uther was unsuccessful in uniting the British warlords towards their continued survival, the impending return of Arthur may be enough to give Dumnonia some pause from the assaults of its enemies - for a time at least."
- BTWK_story_event.0115.desc_norwenna_dead:0 "With Norwenna not long passing after her husband - the late Prince Mordred - Uther's intention of an alliance forged through marriage was not to be, leaving Dumnonia more exposed than he would've liked. However, some loyalists to the High King and his young heir were still to be found for its defence. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. While Uther was unsuccessful in uniting the British warlords towards their continued survival, this little web of alliances will at least give Dumnonia some reprieve from the assaults of its enemies - for a time at least."
+ BTWK_story_event.0115.desc_header:0 "The High King of Britannia and King of [GetTitleByKey('k_dumnonia').GetNameNoTier], Uther Pendragon, has recently hosted a High Council in the old Roman town of Glevum."
+ BTWK_story_event.0115.desc_alliance:0 "Word has reached you that through arranging a marriage between Norwenna - the widow of the late Prince of Dumnonia, Mordred - and [lady_norwenna_husband.GetShortUINameNoFormat], Uther has secured an alliance between his realm, and [dumnonia_alliance_realm.GetNameNoTier]. A number of guardians have also been sworn in by oath to protect the new Edling Mordred. Chiefly among them is [GetTitleByKey('k_gwent').GetHolder.GetShortUINameNoFormat], King of [GetTitleByKey('k_gwent').GetNameNoTier], the legendary druid [GetTitleByKey('c_avalon').GetHolder.GetShortUINameNoFormat], and Uther's bastard son - the renowned warlord Arthur. Such a web of alliances will surely fortify Dumnonia against its enemies until the young Prince reaches maturity."
```

Comment:
1. "between his realm, and" -> "between his realm and"
2. "Chiefly among them..." - I just checked, in the books it's just those 4 who take the oath right? If it is then this isn't really the right wording
3. "the renowned warlord Arthur" -> "the renowned warrior Arthur"

Reply:
1.  Amended.
2. I hadn't included Owain in the event localisation. I may need to add another triggered_desc to account for the fact Tewdric might be dead (assuming Merlin and Arthur will be given health boons to stop them dying before this point). If I am to include Owain in the description (to remove chiefly, will I add triggered_desc's for the event that (i) Tewdric and/or Owain are dead, (ii) player Tewdric and/or Owain do not take the death oath.
3. Amended. But I do believe he is referred to as a warlord in the books.